### ISSUE TITLE
Improve accessibility with ARIA attributes

### ISSUE THREAD

**Issue by slavanga** [cferdinandi/tabby#96]
Tabby should use ARIA attributes to improve accessibility.

Inspiration:
- [ARIA Tab Panel Example](http://accessibility.athena-ict.com/aria/examples/tabpanel2.shtml)
- [Foundation Tabs](https://foundation.zurb.com/sites/docs/tabs.html)

**Comment by cferdinandi** [comment]
While I agree that tabby needs some a11y improvements, and ARIA may certainly be part of that, I'd like to also point out that:

1. Foundation has a *terrible* track record with accessibility, and...
2. ARIA is not strictly necessary for good accessibility.

Digging into point 2 for a minute: generally speaking, when HTML semantics alone provide sufficient accessibility, it's detrimental to also add ARIA. It should be used when and only when HTML and CSS alone do not provide sufficient intent and wayfaring to individuals using assistive devices like screenreaders.

So with that in mind, can you identify any specific aspects of Tabby that are currently inaccessible or do not work for users with assistive devices? If so, I'd be happy to address them.

**Comment by cferdinandi** [comment]
A complete rewrite with proper accessibility, roles, etc. is being released tomorrow

**Comment by cferdinandi** [comment]
Done: https://github.com/cferdinandi/tabby/releases/tag/v12.0.0
